addon/components/lt-infinity.js

```diff
@@ -14,13 +14,17 @@ export default Component.extend(InViewportMixin, {
 
   didInsertElement() {
     this._super(...arguments);
+
+    let scrollBuffer = this.get('scrollBuffer');
```

Comment:
not sure what it's deal is, but there's so many variables that aren't being reassigned, so should be const
https://github.com/dockyard/styleguides/blob/master/engineering/javascript.md#assignment